californium-core/src/main/java/org/eclipse/californium/core/network/CoapEndpoint.java

```diff
@@ -1076,6 +1076,7 @@ public void cancelObservation(Token token) {
 	 *
 	 * @param task the task
 	 */
+
```

Comment:
Please, not source format changes without code changes. So, please revert this change.

Reply:
Of course.